src/js/backend-script.js

```diff
@@ -0,0 +1,31 @@
+function confirmNewUser(e) {
+  const $urlInput = $('#confirmUserURL'),
+    url = $urlInput.val(),
+    $token = $('input[name="csrfmiddlewaretoken"]');
+
+  $.post(
+    url,
+    {
+      csrfmiddlewaretoken: $token.val()
+    }
+  );
+}
+
+function showNewUserModal($) {
+  const $hfShowModal = $('#showNewUserModal'),
+    $newUserModal = $('#newUserModal'),
+    $newUserModalBtn = $('#newUserModalBtn'),
+    showModal = $hfShowModal ? $hfShowModal.val() : false;
+
+  if (showModal === 'True') {
+    $newUserModal.modal();
+  }
+
+  $newUserModalBtn.click(confirmNewUser);
```

Comment:
Another thing to consider: the user can dismiss the introduction modal using the close button, or by clicking outside of the modal, which will not trigger the new user update POST.  Not sure if that makes a difference in functionality throughout the rest of the app or not.